script_editor/app/views/plays/show.html.erb

```diff
@@ -125,6 +125,88 @@
             <!-- This button shows the play's analytics -->
             <button class="nav-analytics" id="analytics-button">ANALYTICS</button>
 
+            <!-- This button toggles the strikethrough edits' visibility -->
+             <!-- Not sure which one to use for display in the else statement, initial or inherit -->
+            <button class = "nav-synopsis" curState = "showing" id="viewToggle-button"> Hide Edits </button>
+
+            <script>
+                var mainScript = document.getElementsByClassName("script-main");
+                var toggleBtn = document.getElementById("viewToggle-button");
+                toggleBtn.curState = "showing";
+                toggleBtn.onclick = function() {
+                    console.log(mainScript);
+                    var words = document.getElementsByClassName("word");
+                    var punctuation = document.getElementsByClassName("punc");
+                    var directions = document.getElementsByClassName("stage");
+                    for (var i = 0; i < words.length ; i++) //toggle words
```

Comment:
Since you never use i as a variable, it may make more sense to use a forEach loop rather than a for loop. These are done by `array.forEach(function)`